certification/internal/policy/operator/deployable_by_olm_test.go

```diff
@@ -0,0 +1,80 @@
+package operator
+
+import (
+	"os"
+	"path/filepath"
+
+	fakecranev1 "github.com/google/go-containerregistry/pkg/v1/fake"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
+)
+
+var _ = Describe("DeployableByOLMCheck", func() {
+	var (
+		deployableByOLMCheck DeployableByOlmCheck
+		engine               cli.OpenshiftEngine
+		imageRef             certification.ImageReference
+	)
+	const (
+		metadataDir        = "metadata"
+		annotationFilename = "annotations.yaml"
+		annotations        = `annotations:
+  operators.operatorframework.io.bundle.package.v1: testPackage
+  operators.operatorframework.io.bundle.channel.default.v1: testChannel
+`
+	)
+
+	BeforeEach(func() {
+		// mock bundle directory
+		tmpDir, err := os.MkdirTemp("", "bundle-metadata-*")
```

Comment:
Still need AfterEach at this level to clean it up.

Reply:
uh gotcha! Good catch!